run-tests.sh

```diff
@@ -3,4 +3,4 @@
 BROWSER=${1:-chrome}
 ENVIRONMENT=${2:-local}
 
-sbt clean -Dbrowser="${BROWSER}" -Denvironment="${ENVIRONMENT}" -Dbrowser.option.headless=false "testOnly uk.gov.hmrc.test.ui.cucumber.runner.Runner" testReport
+sbt clean -Dbrowser="${BROWSER:=chrome}" -Denvironment="${ENVIRONMENT:=local}" "testOnly uk.gov.hmrc.test.ui.cucumber.runner.Runner" testReport
```

Comment:
the current code already has default browser and environment values set. There's no need to hardcode them and leave unused variables like DEFAULT_BROWSER, BROWSER_TYPE, etc.

Also suggest setting `Dbrowser.option.headless=true` instead of removing it, so when we want to test locally, we can simply set it to false.